src/main/java/org/itsallcode/openfasttrace/importer/tag/TagImporterFactory.java

```diff
@@ -21,24 +28,63 @@
  * <http://www.gnu.org/licenses/gpl-3.0.html>.
  * #L%
  */
-
 import org.itsallcode.openfasttrace.importer.*;
 import org.itsallcode.openfasttrace.importer.input.InputFile;
+import org.itsallcode.openfasttrace.importer.tag.config.PathConfig;
+import org.itsallcode.openfasttrace.importer.tag.config.TagImporterConfig;
 
 /**
  * {@link ImporterFactory} for tags in source code files.
  */
 // [impl->dsn~import.full-coverage-tag~1]
-public class TagImporterFactory extends RegexMatchingImporterFactory
+public class TagImporterFactory extends ImporterFactory
 {
-    public TagImporterFactory()
+    private static final Logger LOG = Logger.getLogger(TagImporterFactory.class.getName());
+
+    private static final String DEFAULT_FILE_REGEX = "(?i).*\\.java";
```

Comment:
I think it would be a good idea to list as many popular programming language file extensions as a default here as possible.

Not all from that list but just as an inspiration:
https://www.file-extensions.org/filetype/extension/name/source-code-and-script-files

Reply:
Instead of hard coding a list we need to make this configurable. I created #168 for this.

Reply:
Roger.